src/layers/core/geojson-layer/geojson-layer.js

```diff
@@ -52,14 +52,14 @@ const defaultProps = {
   getElevation: f => 1000
 };
 
-function noop() {}
-
 const getCoordinates = f => get(f, 'geometry.coordinates');
 
 export default class GeoJsonLayer extends Layer {
+
   initializeState() {
     this.state = {
       subLayers: null,
+      subLayerCount: 0,
```

Comment:
This looks confusing, next to `subLayers`? Since we can't use `subLayers.length` I assume `subLayers` is something else (e.g. `subLayerData`). Some cleanup of naming could be helpful.